### ISSUE TITLE
JsonSchema equality comparison ignores additional properties

### ISSUE THREAD

**Issue by desmondgc** [gregsdennis/Manatee.Json#205]
In the following example, `schema2` contains an additional property not found in `schema1`. I would expect Equals to return false but it currently returns true!

```
var serializer = new JsonSerializer();

var schema1 = serializer.Deserialize<JsonSchema>(JsonValue.Parse(@"
{
  ""$schema"": ""http://json-schema.org/draft-07/schema#"",
  ""properties"": {
    ""foo"": { ""type"": ""string"" },
    ""bar"": { ""type"": ""integer"" }
  },
  ""additionalProperties"": false
}"));

var schema2 = serializer.Deserialize<JsonSchema>(JsonValue.Parse(@"
{
  ""$schema"": ""http://json-schema.org/draft-07/schema#"",
  ""properties"": {
    ""foo"": { ""type"": ""string"" },
    ""bar"": { ""type"": ""integer"" },
    ""baz"": { ""type"": ""boolean"" }
  },
  ""additionalProperties"": false
}"));

Console.WriteLine(schema1.Equals(schema2)); // True!
```

**Comment by gregsdennis** [comment]
Thanks for reporting this. I'll have a look. I expect to have an update out soon for the upcoming draft ~8~ 2019-04.

**Comment by gregsdennis** [comment]
@desmondgc, I released 10.2.0-beta1 which contains your fix.
